### FIELD DATA CAN BE ONLY ONE OF THE FOLLOWING ###
google_chat_ros/FormAction action # A form action will be triggered by this onclick if specified
string open_link_url # A link that opens a new window
